chat-stable-beluga-2-process/manifest.json

```
@@ -0,0 +1,49 @@
+{
+    "id": "stable-beluga-2",
+    "serviceType": "process",
+    "version": "1",
+    "name": "Stable Beluga 2",
+    "description": "",
+    "documentation": "",
+    "beta": true,
+    "icon": "",
+    "modelInfo": {
+        "memoryRequirements": 5740
+    },
+    "interfaces": [
+        "chat"
+    ],
+    "defaultPort": 8734,
+    "defaultExternalPort": 8734,
+    "weightsDirectoryUrl": "https://huggingface.co/petals-team/StableBeluga2/resolve/main/",
+    "weightsFiles": [
+        "config.json",
+        "model.safetensors.index.json",
+        "special_tokens_map.json",
+        "tokenizer.model",
+        "generation_config.json",
+        "model_00081-of-00081.safetensors",
+        "tokenizer.json",
+        "tokenizer_config.json"
+    ],
+    "binariesUrl": {
+        "aarch64-apple-darwin": "https://raw.githubusercontent.com/premAI-io/prem-services/dev/cht-petals/setup-petals.sh",
+        "x86_64-apple-darwin": "https://raw.githubusercontent.com/premAI-io/prem-services/dev/cht-petals/setup-petals.sh",
+        "universal-apple-darwin": "https://raw.githubusercontent.com/premAI-io/prem-services/dev/cht-petals/setup-petals.sh"
+    },
+    "userOptions": {
+        "ModelPath": {
+            "type": "str",
+            "help": "Path to locally available model files",
+            "default": "./ml/models--petals-team--StableBeluga2",
+            "choices": null
+        },
+        "DHTPrefix": {
+            "type": "str",
+            "help": "DHT prefix to use",
+            "default": "StableBeluga2-hf",
+            "choices": null
+        }
+    },
+    "serveCommand": "./setup-petals.sh --model_path {ModelPath} --dht_prefix {DHTPrefix} --port {externalPort}"
```

need to revert this for now

```suggestion
    "serveCommand": "./setup-petals.sh --model-path ./ml/models--petals-team--StableBeluga2 --dht-prefix StableBeluga2-hf --port 8734"
```